docs/MapColonies/DEM/Services/elevation/README.md

```diff
@@ -0,0 +1,89 @@
+---
+id: dem-elevation-service
+slug: info
+title: Elevation ⛰️
+description: Service to return geographic coordinate [lat, lon] elevation 
+tags:
+  - dem
+  - API
+  - elevation
+  - height
+  - coordinate
+---
+
+# Elevation Service ⛰️
+
+:::info
+Click [here](/docs/MapColonies/dem/services/elevation/api) for the OpenAPI
+:::
+
+## Purpose
+The service lets users to recieve elevation(height) info at a given location(s)/position(s) [lon,lat] with/without extra data eg. resolution, DTM/DSM, etc.
+
+## Terminology
+
+Elevation info might be returned from following materials:
+1. DTM - terrain materials.
+2. DSM - surface materials.
+3. MIXED - according to most detailed(best resolution) materials.
+
+## Usage
+
+
+:::caution
+**- You can ask elevation values for top 250 points at once. If you need more points to process you should devide it to bulks**
```

Comment:
Please change to:
You may request values for up to 250 points at once. In case you have more than that, you should divide it into bulks.

Reply:
DONE